website/docs/add-secure-apps/providers/saml/IDP-initiated-single-logout.md

```diff
@@ -0,0 +1,28 @@
+---
+title: SAML IDP Initiated Single Logout
+description: Configure SAML IDP Initiated Single Logout for SAML Providers
+authentik_version: "2025.8.0"
+---
+
+authentik supports SAML IDP Initiated Single Logout (SLO), which allows users to log out of authentik and all connected SAML service providers simultaneously.
+In order to use this feature, you must set a [Single Logout Service URL](./index.md#single-logout-service-url) in your SAML Provider settings.
+
+## Logout methods
+
+When users log out of authentik, they are automatically logged out of all SAML service providers accessed during their session. authentik supports three logout methods:
+
+- **Iframe-based front-channel logout** (default) - Performs parallel logout requests using hidden iframes
+- **Redirect-based front-channel logout** - Sequential logout using browser redirects (legacy)
+- **Back-channel logout** - Server-to-server POST requests without user interaction
+
+:::info
+Front-channel logouts occur in the user's browser, while back-channel logouts happen directly between servers and work even when sessions are terminated administratively.
+:::
+
+### Logout stage configuration
```

Comment:
I think this whole section belongs over in the docs on Logout User stage. This page is about IDP-initiated back-channel logout.. was expecting to see confoig steps for that here.